tests/src/main/scala/tapir/tests/package.scala

```diff
@@ -12,6 +12,8 @@ import scala.io.Source
 
 package object tests {
 
+  implicit val fruitAmountCodec: Codec[FruitAmount, MediaType.XWwwFormUrlencoded, _] = ???
```

Comment:
For some reason the implicit is not found on scala 2.13. Maybe someone who knows more about implicit and magnolia and stuff could help here.

Reply:
In `core`s tests, I'm getting a compile error for the tests:

```
[error] /Users/adamw/projects/tapir/core/src/test/scala/tapir/generic/FormCodecDerivationTest.scala:143:27: type mismatch;
[error]  found   : scala.collection.MapView[String,Seq[String]]
[error]  required: Map[String,Seq[String]]
[error]     val codec = implicitly[Codec[Test1, MediaType.XWwwFormUrlencoded, String]]
```

probably sth collection-related in the macro ...

Reply:
Yeah, mapValues returns a MapView, an explicit conversion to Map is needed

In `FormCodecMacros:47`, you'll need to add a `.toMap` at the end of the line:

```
          val paramsMap: Map[String, Seq[String]] = params.groupBy(_._1).mapValues(_.map(_._2)).toMap
```